common-npm-packages/webdeployment-common/msdeployutility.ts

```diff
@@ -113,13 +113,15 @@ export function getMSDeployCmdArgs(webAppPackage: string, webAppName: string, pr
 function escapeQuotes(additionalArguments: string): string {
     const parsedArgs = parseAdditionalArguments(additionalArguments);
     const separator = ",";
-
+    let CommaSeperatedCSEnabled = tl.getPipelineFeature('CommaSeperatedConnectionString');
```

Comment:
Here and on line 123 - please fix naming of local variables - those should start with small letter instead of capital one like it's now